resources/css/index.css

```diff
@@ -0,0 +1,153 @@
+.header-top {
+  padding: 20px 10px 15px;
+  background: #f72210;
+  position: fixed;
+  z-index: 2;
+  top: 0;
+  left: 0;
+  right: 0;
+}
+
+.header-date {
+  color: white;
+  font-size: 16px;
+  font-weight: 400;
+  font-family: "Roboto Condensed", sans-serif;
+  line-height: 34px;
+  padding-left: 20px;
+}
+
+.header-logo {
+  cursor: pointer;
+}
+
+.header-logo svg {
+  fill: white;
+}
+
+.header-login-subscribe {
+  padding-right: 15px;
+}
+
+.btn-yellow {
+  background-color: #ffe400;
+}
+
+.header-links {
+  margin-top: 85px;
+  font-size: 19px;
+  font-weight: 700;
+}
+
+.categories-first {
+  border-left-width: 0;
+}
+
+.header-links-categories ul {
+  margin-bottom: 0;
+}
+
+.header-links-trending ul li a {
+  color: #807f84;
+  text-decoration: none;
+}
+
+.header-links-categories ul li a:hover,
+.header-links-categories ul li a:focus {
+  color: #f72210;
+}
+
+.header-links-trending {
+  padding: 14px 0 14px;
+  margin-top: 4px;
+  border-top: 2px solid rgb(244, 245, 247);
+  border-bottom: 2px solid rgb(244, 245, 247);
+}
+
+.header-links-trending ul {
+  font-size: 17px;
+  font-size: 16px;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-align: center;
+  margin-bottom: 0;
+}
```

Comment:
Avoid using custom styling where bootstrap styling is possible. Example in the above, you can use `text-uppercase`, `text-center`, `mb-0` bootstrap classes. Please check the rest of your styling as well.